src/jquery.ime.inputmethods.js

```diff
@@ -1274,6 +1278,10 @@
 			autonym: 'ಕನ್ನಡ',
 			inputmethods: [ 'kn-transliteration', 'kn-inscript', 'kn-kgp', 'kn-inscript2' ]
 		},
+		kor: {
+			autonym: '한국어',
+			inputmethods: [ 'kor-rr' ]
```

Comment:
We usually give them an identified that is based on the shortest language code, so it should be "ko" and not "kor".